README.md

```diff
@@ -65,7 +66,7 @@ standup-notes -edit --today --copy
 ```
 
 
-#####Delete Notes
+####Delete Notes
```

Comment:
I think you need a space between the last `#` and the start of the title for it to render properly.

Reply:
Put spaces in. 
From the look of other titles they formatted fine without the spaces.